android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -329,7 +334,7 @@ public void processProperties(KrollDict d)
 		TiUIHelper.linkifyIfEnabled(tv, d.get(TiC.PROPERTY_AUTO_LINK));
 		tv.invalidate();
 	}
-	
+
```

Comment:
Did you do the backport manually? I don't see this line in the backport, so I was wondering. Try to cherry-pick all changes for consistency.